source/guides/transfer/sending-an-encrypted-message.rst

```diff
@@ -34,7 +34,7 @@ Prerequisites
 Setting up the use case
 ***********************
 
-Create an account for Alice, and another for the certificate with the :doc:`CLI tool <../../cli>`.
+Create an account for Alice, and another for the certificate with the :ref:`CLI tool <wallet-cli>`.
```

Comment:
Create 2 accounts: one for Alice and another for the certificate with the :ref:`CLI tool <wallet-cli>`.